code/game/objects/items/clown_items.dm

```diff
@@ -111,8 +111,11 @@
 	if(prob(skillcheck*100)) //higher level = more uses assuming RNG is nice
 		uses--
 	if(uses <= 0)
-		to_chat(user, span_warning("[src] crumbles into tiny bits!"))
-		qdel(src)
+		if(!iscyborg(user))
+			to_chat(user, span_warning("[src] crumbles into tiny bits!"))
+			qdel(src)
+		else
+			to_chat(user, span_warning("The soap has ran out of chemicals!"))
```

Comment:
This logic should be handled by a subtype. You'd want to turn the message/destruction behavior into a proc that can be overridden